cmd/cmd.go

```diff
@@ -66,13 +58,67 @@ func init() {
 	rootCmd.Flags().BoolVar(&outputStdOut, "stdout", false,
```

Comment:
Arguably outputting to stdout by default is the UNIX way. But ultimately a judgement call.

Reply:
I'd rather not here because you'll want a file output in all cases, and it complicates error logging a lot. (Splitting stderr and stdout doesnt feel like the Go way to me personally)